docs/en/sql-reference/sql-statements/Data Definition/ALTER TABLE.md

```diff
@@ -71,6 +71,8 @@ under the License.
             1) The following attributes of the modified partition are currently supported.
                 - storage_medium
                 - storage_cooldown_time
+                - storage_cold_medium
```

Comment:
Do not need storage cold medium, If user specified storage resource then it implies the cold storage is S3 or HDFS.

Reply:
There is also a old logic data migration from SSD to HDD, we use both `storage_cold_medium` and `remote_storage_resource` to distinguish them

Reply:
yeah

Reply:
If BE have ssd,hdd,and s3 resource, how to define the operation from ssd->hdd->s3?